CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+#### Bugfixes
```

Comment:
Let's rename this to `Improvements` since it is not a bug. Also, add `## not released yet`  title at the top of this section. We will replace it with `## v1.2.0 - date` (like other release logs) after the release.